src/Components /App/App.js

```diff
@@ -3,17 +3,34 @@ import movieData from '../../Movies-sample.js'
 import Movies from '../Movies/Movies.js'
 
   
-  function App(){
-     
-      const [movies, setMovies] = useState(movieData.movies)
-      
-      return(
-          <main className='App'>
-          <h1>Rancid-Tomatillos</h1>
-          <p>Movie Time!</p>
-          <Movies movies = {movies}/>
-      </main>
-    )
+function App(){
+  const [movies, setMovies] = useState(movieData.movies)
+  const [chosenMovie, setChosenMovie] = useState(null);
```

Comment:
Instead of null, consider setting this data to an empty array or object - whatever for the data will eventuallly take.